Fig6_PyGMT_geopandas.py

```diff
@@ -0,0 +1,38 @@
+import pygmt
+import geopandas as gpd
+
+provider = "https://naciscdn.org/naturalearth/"
+world = gpd.read_file(f"{provider}50m/cultural/ne_50m_admin_0_countries.zip")
+world["POP_EST"] *= 1e-6
+rivers = gpd.read_file(f"{provider}110m/physical/ne_110m_rivers_lake_centerlines.zip")
+cities = gpd.read_file(f"{provider}110m/cultural/ne_110m_populated_places_simple.zip")
+cities_small = cities[cities["worldcity"]!=1]  # Smaller cities
+cities_world = cities[cities["worldcity"]==1]  # Larger (world) cities
+
+fig = pygmt.Figure()
+fig.basemap(region=[-19.5, 53, -38, 37.5], projection="M15c", frame=True)
+pygmt.makecpt(cmap="bilbao", series=(0, 100))

```

Comment:
| series=(0, 270) | series=(0, 100) |
| --- | --- |
| <img width="1915" height="2295" alt="pygmt_paper_geopandas_city_subsets" src="https://github.com/user-attachments/assets/b90d4b89-59e9-41f9-90d1-bab1b6c67659" /> | <img width="1915" height="2295" alt="image" src="https://github.com/user-attachments/assets/e2796206-c9a8-4be4-8ffe-083f6707c343" /> |

Reply:
I'm fine with both with a slight preference for the left one. 

- Left has less color variance but shows the absolute values and makes it easy to compare countries. Shown rivers are more visible since they just run through countries with bright colors.  
- Right has more color variance but comparisons based on raw numbers > 100 are not possible. 

However, since we just want to show what's possible with pygmt, the storytelling itself has not the highest priority here and thus absolute numbers are less important.